contrib/node/install-sdn.sh

```diff
@@ -18,22 +18,20 @@ os::util::install-sdn() {
     popd > /dev/null
   else
     local osdn_base_path="${deployed_root}/Godeps/_workspace/src/github.com/openshift/openshift-sdn"
-    local osdn_controller_path="${osdn_base_path}/pkg/ovssubnet/controller"
-    pushd "${osdn_controller_path}" > /dev/null
-      # The subnet plugin is discovered via the kube network plugin path.
-      local kube_osdn_path="${target}/libexec/kubernetes/kubelet-plugins/net/exec/redhat~openshift-ovs-subnet"
-      mkdir -p "${kube_osdn_path}"
+    local osdn_plugin_path="${osdn_base_path}/plugins/osdn"
+    pushd "${osdn_plugin_path}" > /dev/null
       mkdir -p "${target}/bin/"
-      cp -f kube/bin/openshift-ovs-subnet "${kube_osdn_path}/"
-      cp -f kube/bin/openshift-sdn-kube-subnet-setup.sh "${target}/bin/"
 
-      # The multitenant plugin only needs to be in PATH because the
-      # origin multitenant plugin knows how to discover it.
+      # Plugins only need to be in PATH because origin and the plugins know how
+      # to find their executables
```

Comment:
With the old comment about the subnet plugin removed, this comment now seems to be saying "The only reason plugins need to be in PATH is because origin and the plugins know how to find their executables", which doesn't make any sense. I think you can just remove the comment entirely.